fluentgen/src/main/java/com/azure/autorest/fluent/template/FluentClientMethodTemplate.java

```diff
@@ -242,7 +242,7 @@ protected void generateLongRunningBeginAsync(ClientMethod clientMethod, JavaType
         boolean mergeContextParameter = settings.getAddContextParameter() && (settings.isContextClientMethodParameter() && contextInParameters(clientMethod));
         String contextParam = mergeContextParameter ? "context" : "Context.NONE";
 
-        typeBlock.annotation("ServiceMethod(returns = ReturnType.SINGLE)");
+        typeBlock.annotation("ServiceMethod(returns = ReturnType.LONG_RUNNING_OPERATION)");
```

Comment:
do we need to update LRO related  tests?

Reply:
No, it is not used by customer. Either it is only informative, or used by azure-core reflex.